packages/grafana-ui/src/components/Combobox/filter.ts

```diff
@@ -0,0 +1,23 @@
+import { ComboboxOption } from './Combobox';
+
+export function itemToString<T extends string | number>(item?: ComboboxOption<T> | null) {
+  if (!item) {
+    return '';
+  }
+  if (item.label?.includes('Custom value: ')) {
+    return item.value.toString();
+  }
+  return item.label ?? item.value.toString();
+}
+
+export function itemFilter<T extends string | number>(inputValue: string) {
+  const lowerCasedInputValue = inputValue.toLowerCase();
```

Comment:
same non blocking comment here. Should we use the fuzzy text matching at some point?

Reply:
At some point, yes.